api/Service/UserService.cs

```diff
@@ -35,6 +34,15 @@ public async Task<IList<User>> GetUserList()
 			return userList;
 		}
 
+		public async Task<IList<User>> GetParticipantsForEvent(int eventId){
```

Comment:
This is a duplicate of        Task<IList<Attendance>> GetParticipants(int EventId); from AttendanceService. We do not need it twice.